collect-settings.rst

```diff
@@ -175,13 +175,60 @@ Admin settings allow you to :ref:`restrict which General Settings are seen by us
 Import/Export settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:guilabel:`Import/Export Settings` lets you configure other devices identical to your current device simply by scanning the QR code. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When scanned by the ODK Collect app with another device, these settings are imported.
+When you click on the :guilabel:`Import/Export Settings`, you see a QR Code and a few options. 
 
-You can also import settings from a QR code saved to the device, by selecting :guilabel:`Select Code from SD Card`
+.. image:: /img/collect-settings/import-settings.*
+  :alt: Import/export settings menu of ODK Collect
+  :class: device-screen-vertical
 
-.. warning:: 
+QR Code
+""""""""
+
+QR Code or the Quick Response Code is a two dimensional barcode. QR codes can be used to configure Collect on many devices. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When this QR code is scanned from the ODK Collect app on another device, these settings are imported.
+
+Sharing QR code
+""""""""""""""""
+
+You can click on the |share| icon to share the generated barcode as an image. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
+
+.. |share| image:: /img/collect-settings/share-icon.*
+             :alt: Share icon for sharing the QR code. 
+             :height: 43 px
+             :width: 43 px
+
+Saving QR code locally
+""""""""""""""""""""""""
+
+You can go to :menuselection:`⋮ --> Save settings to disk` to save the QR code.  
 
-  The QR code used for settings import contains the admin and server passwords *in plain text*. To remove them from the code, :gesture:`tap` the warning on the QR code screen.
+Making your own QR code
+""""""""""""""""""""""""""
+
+You can import settings from a QR code saved on your device by selecting :guilabel:`Select Code from SD Card` option. The code that you select could be something that was shared with you or you can also generate your own QR code. To create your own QR code, you need to know that QR code is a JSON object with a structure as shown below:
+
+.. code-block:: JSON
+
+  {
+    "general": {
+      "protocol": "google_sheets",
+      "constraint_behavior": "on_finalize"
+    },
+    "admin": {
+      "edit_saved": false
+    }
+  }
+
```

Comment:
In addition to the sample code here,
the issue #107 requires a complete list of all the settings.
This is needed because there is currently no way for an implementer to know how everything is named.

This list would not be in sample code, because it needs to include the possible options for each setting name. There might be an appropriate text role or directive in the Sphinx Javascript domain, or you may just need to use a definition list.